### DISCUSSION TITLE
Todo mode keyboard shortcut

### DISCUSSION THREAD

**Discussion by jaronheard** [rxhanson/Rectangle#336] [Ideas]
Todo mode is awesome! It would be awesome to be able to toggle Todo mode on and off with a keyboard shortcut.

**Comment by rxhanson** [comment]
I was thinking the same thing. I would like to add a recorder for the reflow shortcut to a new UI tab at some point, and that would be a good time to add this in as well.

**Reply by aakashb95** [reply]
I was wondering if Todo app width could be customised to fit into the last fourth of the screen. I use microsoft todo app and default todo width is slightly wide.

**Reply by rxhanson** [reply]
You can adjust the width of the todo application with the following command and an app restart:

```bash
defaults write com.knollsoft.Rectangle todoSidebarWidth -float 400
```

**Reply by aakashb95** [reply]
Excellent. Thank you so much!
